src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -75,7 +76,27 @@ public void start() {
 			appWindows.checkAndShowUpdateReminderWindow();
 		}
 
+		handleDokanySupportEndAndShowDialog();
+
 		launchEventHandler.startHandlingLaunchEvents();
 		autoUnlocker.tryUnlockForTimespan(2, TimeUnit.MINUTES);
 	}
+
+	private void handleDokanySupportEndAndShowDialog() {
```

Comment:
This method name is bit much (long, confusing, explicit).

I suggest to make it easier to read by renaming it to ( `migrateAndInformDokanyRemoval()`